src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -53,11 +53,35 @@ class PhpDocExtractor implements PropertyDescriptionExtractorInterface, Property
      */
     private $phpDocTypeHelper;
 
-    public function __construct(DocBlockFactoryInterface $docBlockFactory = null)
+    /**
+     * @var string[]
```

Comment:
docblocks are actually not needed when private properties are initialised in the constructor